icubam/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.1.2'
```

Comment:
Maybe this should be kept in the setup.py, and it will get pushed on pip install.

Reply:
Yes, in the end it should be. I opened https://github.com/icubam/icubam/issues/121 for it.

I'll open a follow up PR to address it.